greedy algorithms/fractional_knapsack.py

```diff
@@ -0,0 +1,73 @@
+"""
+Author : Robin Singh
+Implementation Of Fractional Knapsack
+Given weights and values of n items, we need to put these items in a knapsack of capacity W to get the maximum total value in the knapsack
+n Fractional Knapsack, we can break items for maximizing the total value of knapsack
+This problem in which we can break an item is also called the fractional knapsack problem
+An efficient solution is to use Greedy approach. The basic idea of the greedy approach is to calculate
+the ratio value/weight for each item and sort the item on basis of this ratio. Then take the item with
+the highest ratio and add them until we can’t add the next item as a whole and at the end add the next item as much as we can
+Which will always be the optimal solution to this problem
+Time Complexity  : O(nLogn)
+"""
+import operator
+
+
+def fractional_knapsack(p, w, capacity):
+
+    n = len(p)
+    new_list = []
+    for i in range(n):
+        new_list.append([p[i], w[i], p[i]*1.0/w[i]])
+
+    new_list = sorted(new_list, reverse=True, key=operator.itemgetter(2))
+    max_profit = 0
+    fractional_val = 0
+    table_profit = []
+    table_weight = []
+    ratio_table = []
+
+    for i in range(n):
+        if capacity > 0 and new_list[i][1] < capacity:
+            capacity -= new_list[i][1]
+            max_profit += new_list[i][0]
+            table_profit.append(new_list[i][0])
+            table_weight.append(new_list[i][1])
+            ratio_table.append(new_list[i][2])
+
+        else:
+            fractional_val = i
+            table_profit.append(new_list[i][0])
+            table_weight.append(new_list[i][1])
+            ratio_table.append(new_list[i][2])
+
+            break
+
+    if capacity > 0:
+        max_profit += capacity * \
```

Comment:
**\** cause error

Reply:
@rcoder23 done

Reply:
@robin025 Not resolve still shows error

Reply:
@rcoder23  I have maded the required changes